:pencil2: fixed get me in mail sending feature and updated tokens

DhruvDave12 opened the pull request:
Fixed the bug because of which mail weren't being send to the users and removed slack feature

neeraj3029 commented on constants.js:
```diff
@@ -2,5 +2,5 @@ exports.token = process.env.SECRET
 exports.webhookURL = process.env.INVITE_CHANNEL_WEBHOOK
 exports.glitch = process.env.GLITCH_SECRET
-exports.selfEmail = 'IIITV Coding Club <codingclub@iiitv.ac.in>'
+exports.selfEmail = 'IIITV Coding Club <davedhruv1201@gmail.com>'
```
change it to CC's email

neeraj3029 commented on public/client.js:
```diff
@@ -81,16 +81,26 @@ $.put = function (url, data, callback, type) {
+        console.log("OUTPUT OF GITHUB API: ", out);
+        var flag = false;
+
+        for(let i=0; i<out.total_count; i++){
```
Can you share a sample response here? Has the response type changed from object to object[] ?

aashutoshrathi replied:
yup! seems like that
https://api.github.com/search/users?q=linus

aashutoshrathi replied:
Also, I think in our case, we don't need to iterate, ideally, there should be either 0 or 1 entries in the response.

aashutoshrathi replied:
Wait I'm not even sure if we need search API, I think our use case will be solved using User API
Eg: https://api.github.com/users/aashutoshrathi

DhruvDave12 replied:
okay sir i will update the right now i checked on entering a username an array of object with username in different combination of lowercase and uppercase were coming so i will update the API to get user one

neeraj3029 commented on public/client.js:
```diff
@@ -81,16 +81,26 @@ $.put = function (url, data, callback, type) {
+        console.log("OUTPUT OF GITHUB API: ", out);
+        var flag = false;
+
+        for(let i=0; i<out.total_count; i++){
```
Use `Array.filter` to filter out required users and remove all the flag logic

DhruvDave12 replied:
yes sir sure will update the code

neeraj3029 commented on public/client.js:
```diff
@@ -109,20 +119,56 @@ $(function () {
+            })
+          }
         }
+        // if (out.total_count === 1) {
```
Remove commented code and logs

neeraj3029 commented:
Why do we have lockfile changes without `package.json` updates?

aashutoshrathi reviewed (changes requested):
There are some comments by @neeraj3029 that needs to be resolved, plus code-quality is bad in general in this repo, so when you get time let's clean that up too.
Once all that is done, let me know I can provide one of you people the access to Glitch deployment (ideally someone with access to CC Email, since it has details linked to that in env)

DhruvDave12 commented:
I have updated the github api to fetch the user directly, removed the comments and unnecessary logs, and also fixed one toast issue that send the message of "Thank you, check your inbox" only if the username exists.

Remaining changes like
1. Updating the CC mail.
2. Putting the credentials of CC.
3. Improving further code quality

This all will be done latest by tomorrow.

Im-Madhur-Gupta reviewed (approved):
LGTM
